scripts/check_odoo_patterns.sh

```
@@ -152,16 +152,16 @@ for py_file in $(echo "$PY_FILES" | grep '_inherit\.py$'); do
     CONTENT=$(cat "$py_file" 2>/dev/null | grep -v '^#' | grep -v '^$' | grep -v 'from\|import' || true)
     LINE_COUNT=$(echo "$CONTENT" | wc -l | tr -d ' ')
     # If file has < 5 non-empty, non-import lines, it's likely empty
-    if [ "$LINE_COUNT" -lt 5 ]; then
+    if [[ "$LINE_COUNT" -lt 5 ]]; then
         # Check if it only has _inherit and nothing else meaningful
         HAS_FIELDS=$(echo "$CONTENT" | grep -E '^\s+\w+\s*=\s*fields\.' || true)
         HAS_METHODS=$(echo "$CONTENT" | grep -E '^\s+def\s+' || true)
-        if [ -z "$HAS_FIELDS" ] && [ -z "$HAS_METHODS" ]; then
+        if [[ -z "$HAS_FIELDS" ] && [ -z "$HAS_METHODS" ]]; then
```

Shellcheck (0.11.0)

[error] Test expression was opened with double [[ but closed with single ]. Make sure they match.

(SC1033)